ColorPicker/Mouse/MouseInfoProvider.cs

```diff
@@ -12,29 +12,43 @@ namespace ColorPicker.Mouse
 {
     [Export(typeof(IMouseInfoProvider))]
     [PartCreationPolicy(CreationPolicy.Shared)]
-    public class MouseInfoProvider : IMouseInfoProvider
+    public sealed class MouseInfoProvider : IMouseInfoProvider, IDisposable
     {
         private const int MousePullInfoIntervalInMs = 10;
         private readonly DispatcherTimer _timer = new DispatcherTimer();
         private readonly MouseHook _mouseHook;
         private readonly IUserSettings _userSettings;
+        
         private System.Windows.Point _previousMousePosition = new System.Windows.Point(-1, 1);
         private Color _previousColor = Color.Transparent;
         private bool _colorFormatChanged = false;
+        
+        private readonly Bitmap _bmp; // 1x1px-sized bitmap to capture desktop pixel via GDI.
+        private readonly Graphics _bmpGraphics;
 
         [ImportingConstructor]
         public MouseInfoProvider(AppStateHandler appStateMonitor, IUserSettings userSettings)
         {
+            _bmp = new Bitmap(1, 1, PixelFormat.Format32bppArgb);
+            _bmpGraphics = Graphics.FromImage(_bmp);
+            
+            _mouseHook = new MouseHook();
```

Comment:
Reordering these because all fields should be initialized _before_ setting-up any event-handlers